### PR TITLE
Brush up Crystal Keeper and Edgewatch Detective archetypes

### PR DESCRIPTION
**Pull request by pedrogrullada**
- Edgewatch Detective
   - Add prerequisites, skill proficiency upgrade and Track with Perception action button to Edgewatch Detective Dedication
   - Add Will save link to Bolera's Interrogation and condense its effects (will add to the tracking issue if merged)
- Crystal Keeper
   - Add predicated custom resistance to Crystal Keeper Dedication
   - Add spell links to Simple Crystal Magic
   - Add item links to Armor Rune Shifter and Weapon Rune Shifter

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -2225,6 +2225,10 @@
                     "Label": "Critical Effect"
                 }
             },
+            "CrystalKeeper": {
+                "RollOptionLabel": "Damage from hazards associated with crystals",
+                "ResistanceLabel": "crystal hazards"
```
Comment (TikaelSol) [inline static/lang/re-en.json:2230]: Move to be with its friends in custom IWR labels

File: packs/feats/edgewatch-detective-dedication.json [hunk 2]
```diff
@@ -26,15 +30,283 @@
...
+                            },
+                            {
+                                "gte": [
+                                    "skill:thievery:rank",
+                                    1
+                                ]
+                            }
+                        ],
+                        "value": "arcana"
+                    },
```
Comment (TikaelSol) [inline packs/feats/edgewatch-detective-dedication.json:70]: What is happening here?

We shouldn't do anything with the "if you're already trained pick a skill" thing, we'll figure out some clever way to handle that when we do skill training management

**Comment by TikaelSol** [comment]
Sorry, hit you with some merge conflicts there

**Comment by pedrogrullada** [comment]
Resolved the conflicts, good to go

File: static/lang/re-en.json [hunk 3]
```diff
@@ -601,6 +601,7 @@
                 "AllStrikeDamageFromUndeadCreatures": "all damage from Strikes made by undead creatures",
                 "AttacksWaterCreatures": "attacks by water creatures",
                 "ColdIronBludgeoning": "cold iron bludgeoning",
+                "CrystalKeeper": "crystal hazards",
```
Comment (TikaelSol) [inline static/lang/re-en.json:604]:
```suggestion
                "CrystalHazards": "crystal hazards",
```

File: packs/feats/crystal-keeper-dedication.json [hunk 4]
```diff
@@ -39,6 +39,24 @@
...
+                "label": "PF2E.SpecificRule.CrystalKeeper.RollOptionLabel",
+                "option": "crystal-hazard",
+                "toggleable": true
+            },
+            {
+                "definition": [
+                    "origin:type:hazard"
+                ],
+                "key": "Resistance",
+                "label": "PF2E.IWR.Custom.CrystalKeeper",
```
Comment (TikaelSol) [inline packs/feats/crystal-keeper-dedication.json:54]:
```suggestion
                "label": "PF2E.IWR.Custom.CrystalHazards",
```

**Pull request merged**